components/post_view/post_image/post_image.jsx

```diff
@@ -103,9 +115,15 @@ export default class PostImageEmbed extends React.PureComponent {
     };
 
     render() {
+        const imageDimentions = getFileDimentionsForDisplay(this.props.dimentions, MAX_IMAGE_DIMENTIONS);
         if (this.state.errored || !this.state.loaded) {
-            // scroll pop could be improved with a placeholder when !this.state.loaded
-            return null;
+            return (
+                <div style={{...imageDimentions, marginBottom: '13px'}}>
```

Comment:
No it is really specific because it was added on the image div class.

I can create if you have a strong opinion on it but otherwise i rather leave it in place as it just adds another class with few more overrides